resources/frequently-asked-questions/README.md

```diff
@@ -122,50 +122,50 @@ Add the following rule to your custom ProGuard rules file, typically labeled `pr
 
 ### How can I track user engagement of push notifications using the Experience Platform Mobile SDK?
 
-How you implement push notification tracking and measurement with the SDK depends on the Experience Cloud solution being used. Specifically, for:
+Implementing push notification tracking and measurement with the SDK depends on the Experience Cloud solution being used.
 
-* Adobe Campaign Standard extension, see [Adobe Campaign Standard Push Tracking](https://helpx.adobe.com/campaign/kb/push-tracking.html).
-* Adobe Campaign Classic extension, see [Adobe Campaign Classic Push Notifications Tracking](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference#tracknotification-api).
-* Adobe Analytics - Mobile Services extension, see [Set Up Tracking for Mobile Services Push Notifications](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics-mobile-services#set-up-push-tracking).
+* For the Adobe Campaign Standard extension, please read the [Adobe Campaign standard push tracking tutorial](https://helpx.adobe.com/campaign/kb/push-tracking.html).
```

Comment:
do we need to start using `please read` for all links? or can we still use ..., see <link> in some instances? I feel the previous `... see <link>` usage was easier/quicker to read.

Reply:
It's not 100% necessary, it's just I prefer it (probably because I like using please as a Canadian).